Source/WebCore/dom/ScriptElement.h

```diff
@@ -52,7 +52,7 @@ class ScriptElement {
 
     bool prepareScript(const TextPosition& scriptStartPosition = TextPosition());
 
-    String scriptCharset() const { return m_characterEncoding; }
+    AtomString scriptCharset() const { return m_characterEncoding; }
```

Comment:
Ideally would return `const AtomString&` to not force the caller to copy.

Reply:
Oh, good point. I think that's how it's done elsewhere, too.